<?xml version="1.0"?>
<launch>
  <rosparam file="$(find template_pkg)/config/parameters.yaml" command="load"/>
  <node name="example_node" pkg="template_pkg" type="example_node" output="screen"/>
</launch>
